src/pages/MyGardens/index.tsx

```diff
@@ -52,7 +72,22 @@ export const MyGardens = () => {
       exit={{ opacity: 0 }}
     >
       <div className="my-gardens-container">
-        <h1>My Gardens</h1>
+        <Grid
```

Comment:
If we are using grid, we can consider making it reactive and scale to a 2 columns on desktop or something

Reply:
The grid component is a flex container, I think we need to make desktop view better in general, but I don't think we need extra columns for this content.